app/controllers/crime_applications_controller.rb

```diff
@@ -9,38 +10,31 @@ def index
                     .merge(Applicant.with_name)
   end
 
+  def create
+    initialize_crime_application do |crime_application|
+      redirect_to edit_steps_client_has_partner_path(crime_application)
+    end
+  end
+
   def edit
     @tasklist = TaskList::Collection.new(
-      view_context, crime_application: crime_application
+      view_context, crime_application: current_crime_application
     )
   end
 
   def destroy
-    @application.destroy
+    current_crime_application.destroy
     redirect_to crime_applications_path,
                 flash: {
-                  success: t('.success_flash', full_name: @full_name)
+                  success: t('.success_flash', applicant_name: @applicant_name)
                 }
   end
 
-  def confirm_destroy
-    render :confirm_destroy
-  end
+  def confirm_destroy; end
```

Comment:
hehe I was called confirm delete to begin with - forgot to delete this line after changing the partial to match the action